src/NodaTime.Web/Startup.cs

```diff
@@ -96,7 +96,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 .AddRedirect($@"^{anyVersion}/userguide$", "$1/userguide/")
                 .AddRedirect($@"^developer$", "developer/")
                 // Make /api and /userguide links to the latest stable release.
-                .AddRedirect("^(api|userguide)((?:/.*))$", "1.3.x/$1$2");
```

Comment:
I'm not sure why I made it greedy at all. I just needed it to be optional. Basically I experimented until it worked :)